sirepo/auth/basic.py

```diff
@@ -38,9 +38,13 @@ def _check(v):
     return cfg.uid == v.username and cfg.password == v.password
 
 
-def init_apis(*args, **kwargs):
+def _init():
```

Comment:
this idiom is probably not useful here since this is just initializing cfg so could remove the function and just cfg = pkconfig. By having an `_init`, it begs the question where is it called. Without it, there's no question.

Reply:
c1482e38